stable/dex/values.yaml

```
@@ -3,9 +3,9 @@
 # Declare name/value pairs to be passed into your templates.
 # name: value
 
-image: docker.io/mesosphere/dex
-imageTag: "v2.35.1-d2iq.1"
-imagePullPolicy: "IfNotPresent"
+-image: docker.io/mesosphere/dex
+-imageTag: "v2.37.0-d2iq-1"
```

```suggestion
-imageTag: "v2.37.0-d2iq.1"
```

Did the tag format changed?